region/maridia/inner-pink.json

```diff
@@ -985,33 +516,23 @@
             {
               "id": 2,
               "requires": [
-                {
-                  "or": [
-                    {
-                      "and": [
-                        "canSwim",
-                        {
-                          "or": [
-                            "canMochtroidClip",
-                            "canPuyoClip",
-                            "canDash"
-                          ]
-                        }
-                      ]
-                    },
-                    {
-                      "and": [
-                        "canSuitlessMaridia",
-                        {
-                          "or": [
-                            "canMochtroidClip",
-                            "canPuyoClip"
-                          ]
-                        }
-                      ]
-                    }
-                  ]
-                },
+                {"or": [
+                  {"and": [
+                    "canSwim",
+                    {"or": [
+                      "canMochtroidClip",
+                      "canPuyoClip",
+                      "canDash"
+                    ]}
+                  ]},
+                  {"and": [
+                    "canSuitlessMaridia",
+                    {"or": [
+                      "canMochtroidClip",
+                      "canPuyoClip"
+                    ]}
+                  ]}
+                ]},
                 "canSwim"
```

Comment:
This looks like a leftover that shouldn't be there?

Reply:
I didn't change the logic here this time around but yeah, that doesn't look right. Good catch.